src/pages/GroupAdmin/components/MembersHistoryCard.tsx

```diff
@@ -0,0 +1,67 @@
+import { useMemo } from 'react';
+
+import { ListItem, ListItemText, ListItemAvatar, List, Grid, Typography, Skeleton } from '@mui/material';
+import { parseISO } from 'date-fns';
+
+import Paper from 'components/layout/Paper';
+import { useMembershipHistories } from 'hooks/Membership';
+import Pagination from 'components/layout/Pagination';
+import { useGroup } from 'hooks/Group';
+import Avatar from 'components/miscellaneous/Avatar';
+import { getMembershipType } from 'utils';
+
+export type MembersHistoryCardProps = {
+  slug: string;
+};
+
+const MembersHistoryCard = ({ slug }: MembersHistoryCardProps) => {
+  const { data, hasNextPage, fetchNextPage, isLoading, isFetching } = useMembershipHistories(slug);
+  const prevMembers = useMemo(() => (data !== undefined ? data.pages.map((page) => page.results).flat(1) : []), [data]);
+  const { data: group } = useGroup(slug);
+  const leader = group?.leader;
+
+  if (isLoading) {
+    return (
+      <Paper>
+        <Skeleton height='100px' variant='rectangular' />
+      </Paper>
+    );
+  }
+
+  if (!data?.pages?.length && !leader) {
+    return null;
+  }
+
+  const formatter_month = new Intl.DateTimeFormat('no-NO', { month: 'long' });
```

Comment:
Bruk `getMonth()` i utils istedenfor denne

Reply:
Brukte formatter her fordi jeg tenkte vi ønsket fulle navn på månedene. Utils metoden gir bare kortversjoner av månedene. Tenker du vi heller skal bruke kortversjon enn fullversjon av månedsnavnene? Kan eventuelt lage en en til funkjson i utils for fullemåneds navn?

Reply:
Tenker kortversjon av månedene er helt greit